tests/nested/manual/uc20-smoke/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Run a smoke test on UC20 with encryption enabled
+
+description: |
+    This test checks if UC20 passes the smoke test suite with secure boot
```

Comment:
IMHO it would be good to enhance this description and say that spread is run again for the smoke tests.

Reply:
fixed